ops/prod/app_gateway_url_redirects.tf

```diff
@@ -11,6 +11,7 @@ locals {
   http_listener                = "${local.name}-http"
   https_listener               = "${local.name}-https"
   frontend_config              = "${local.name}-config"
+  zones                        = ["1", "2", "3"]
```

Comment:
this this also use `local.zones`

Reply:
This sets `local.zones` for the rest of the file, without this, `local.zones` doesn't exist.